app/domain/authentication/installed_authenticators.rb

```diff
@@ -33,20 +33,20 @@ def configured_authenticators
           .push(::Authentication::Common.default_authenticator_name)
       end
 
-      def enabled_authenticators(env)
+      def enabled_authenticators
         # Enabling via environment overrides enabling via CLI
-        env_enabled_authenticators(env) || db_enabled_authenticators
+        env_enabled_authenticators || db_enabled_authenticators
       end
 
-      def enabled_authenticators_str(env)
-        enabled_authenticators(env).join(',')
+      def enabled_authenticators_str
+        enabled_authenticators.join(',')
       end
 
       private
 
-      def env_enabled_authenticators(env)
-        authenticators = env["CONJUR_AUTHENTICATORS"]
-        authenticators.present? ? authenticators.split(',') : nil
+      def env_enabled_authenticators
```

Comment:
I would call this `configured_authenticators` now.

Reply:
configured_authenticators is already used

Reply:
Hah, okay. In that case (and knowing that we're going to tear out the DB-configured authenticators sooni-ish) I would get rid of this helper method and just build the logic into the `enabled_authenticators` method.